README.md

```diff
@@ -1 +1,57 @@
-# coding-coach-back-end
\ No newline at end of file
+# Coding-coach GraphQL API
+
+This is the Coding-coach backend project.
+
+### Installing and running
+
+Just clone this repo and run
+
+```
+npm install
+```
+
+or
+
+```
+yarn
+```
+
+Starting the server
+
+```
+npm start
+```
+
+or
+
+```
+yarn start
+```
+
+The GraphQL endpoint will run at http://localhost:4000/graphql
+
+### Docker
+
+To build the docker image just run:
+
+```
+docker build -t code-coaching-api .
+```
+
+To run the container:
+
+```
+docker run -pYOUR_PORT:4000 code-coaching-api
```

Comment:
Most of this looks great! If I had to nitpick it'd be:

1. should there be a space after `-p` ?
2. probably overkill, but maybe notes telling other devs when to use docker and how to stop images? or just links to documentation for them because I think there might still be a good number of people who haven't used docker yet?